commands/user_e2e_test.go

```diff
@@ -136,3 +138,63 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestListUserCmd() {
+	s.SetupTestHelper().InitBasic()
+			
+	// populate map for checking
+	userPool := map[string]bool{
+		s.th.BasicUser.Username : true,
+		s.th.BasicUser2.Username: true,
+		s.th.TeamAdminUser.Username:true,
+		s.th.SystemAdminUser.Username:true,
+	}
+	for i := 0; i < 10; i++ {			
+		usr := s.th.CreateUser()
+		userPool[usr.Username] = true
+	}
+
+	s.RunForAllClients("Get some random of user", func(c client.Client) {
+		printer.Clean()
+
+		page:= int(0)
+		perpage:= int(5)
+		all := bool(false)		
```

Comment:
No need to cast or even assign for zero values.
```suggestion
		var page int
		perpage := 5
		var all bool
```